frontend/src/global_styles/content/work_packages/inplace_editing/_textareas.sass

```diff
@@ -50,8 +50,7 @@
   width: 80px
   height: 40px
   background: var(--control-bgColor-rest)
-  border: 1px solid var(--inplace-edit--color--very-dark)
-  box-shadow: 2px 2px 4px var(--inplace-edit--border-color)
```

Comment:
Is there a reason you removed the shadow effect?

Reply:
Hi @HDinger 
inplace-edit--controls are two buttons at the bottom of text area, I thought that would be more consistent with other buttons if I remove the box-shadow. Buttons don't have box-shaow, they have only border.

Reply:
I don't think that these buttons are normal buttons as they are more form-field controls. However, I am fine with leaving it like it is now. 👍